terraform/modules/tempo/outputs.tf

```diff
@@ -13,10 +13,38 @@ output "app_names" {
   )
 }
 
+output "requires" {
+  value = {
+    logging           = "logging",
+    ingress           = "ingress",
+    certificates      = "certificates",
+    send-remote-write = "send-remote-write",
+  }
+}
+
+output "provides" {
+  value = {
+    grafana_dashboard = "grafana-dashboard",
+    grafana_source    = "grafana-source",
+    metrics_endpoint  = "metrics-endpoint",
+    tracing           = "tracing",
+  }
+}
+
```

Comment:
See "endpoints" below. We [moved](https://github.com/canonical/observability/pull/226) to that notation to mimic juju closer: when an admin does `juju relate`, they don't need to know which is the require and which is the provider.

Let's refactor "requires" and "provides" into "endpoints"?